site/source/docs/contracts_configuration.md

```diff
@@ -450,13 +450,13 @@ deploy: {
 ...
 ```
 
-Notice how `dependencies.contracts` gives access to the `Manager` contract instance. This however, is only possible because `Manager` has been defined as dependency of `ERC20` using the `deps` property. If we're using a Node version that doesn't support async/await, the same can be achieved using promises like this (web3 APIs already return promises):
+Notice how `contracts` gives access to the `Manager` contract instance. This however, is only possible because `Manager` has been defined as dependency of `ERC20` using the `deps` property. If we're using a Node version that doesn't support async/await, the same can be achieved using promises like this (web3 APIs already return promises):
 
 ```
 ...
 ERC20: {
-  deployIf: (dependencies) => {
-    return dependencies.contracts.Manager.methods.isUpdateApproved().call();
+  deployIf: ({contracts, web3, logger}) => {
```

Comment:
This example shouldn't be needed if we drop the sentence about Node versions that don't support async/await.

Reply:
done